Add September Relnotes (#30)
Axway/amplify-central: content/en/docs/amplify_relnotes/20210930_ampc_agents_relnotes.md

---
title: Amplify agents September 2021 Release Notes
linkTitle: Amplify agents September 2021
weight: 90
date: 2021-09-30
description: Traceability and Discovery agents for Amplify Gateway / AWS / Azure
  / Istio provide better visibility into your multi-type gateway eco system.
  These agents collect data from the Gateway (API / traffic) and expose it in
  Amplify Central, providing you with a global vision of your eco system from a
  single interface.
---

## Versioning

Currently, version 1.1.4 is available. It is based on Amplify Agents SDK v1.1.7.
To display version information in the agents, use command `agentName --version`.

This version is compatible with:

* Axway API Management 7.6.2 SPx and 7.7 SPx
* AWS Gateway using SDK 2.0
* Azure latest release

## New features and enhancements

The following new features and enhancements are available in this update.

### Amplify Gateway agents enhancements

> * **Categorization of your API**: To facilitate the API search in the Unified Catalog using the API grouping per category, the discovery agent has been enhanced to take advantage of the tagging done on the API Manager. The configuration of the agent allows to map a tag from API Manager to one category or multiple in Unified Catalog. If the category is not yet existing, another configuration allow the agent to automatically create it (by default missing category are not created). For more information, go to [Use Discovery Agent to categorize your APIs](/docs/connect_manage_environ/connected_agent_common_reference/category_mapping).

Author: ```suggestion
* **Categorization of your API**: To facilitate the API search in the Unified Catalog using the API grouping per category, the Discovery Agent has been enhanced to take advantage of the tagging done in the API Manager. The configuration of the agent allows mapping a tag from API Manager to one or multiple categories in Unified Catalog. If a category does not exist, the agent can be configured to automatically create it (by default, missing categories are not created). For more information, go to [Use Discovery Agent to categorize your APIs](/docs/connect_manage_environ/connected_agent_common_reference/category_mapping).
```
